include/engine/datafacade/contiguous_internalmem_datafacade.hpp

```diff
@@ -154,7 +155,7 @@ class ContiguousInternalMemoryAlgorithmDataFacade<algorithm::CoreCH>
     : public datafacade::AlgorithmDataFacade<algorithm::CoreCH>
 {
   private:
-    util::ShM<bool, true>::vector m_is_core_node;
+    util::ShM<unsigned, osrm::storage::MemorySetting::SharedMemory>::vector m_is_core_node;
```

Comment:
Sure, compilation kept failing on my computer here until I changed to `unsigned`. I'll look into re-introducing `bool` and figuring out another way to proceed.